app_server/users.py

```diff
@@ -27,17 +27,17 @@ def _request_friendships(user_email, new_friends_email):
 												new_friends_email,
 												client[DB][coll])
 	if status_code == HTTP_CREATED:
-		message = 'Your request was completed successfully'
+		message = 'La solicitud fue completada con éxito'
 	elif status_code == HTTP_FORBIDDEN:
-		message = 'This user is already your friend!'
+		message = 'Este usuario ya es tu amigo'
 	elif status_code == HTTP_NOT_FOUND:
-		message = 'The request could not complete successfully because one of the users is not valid'
+		message = 'La solicitud no se pudo completar porque uno de los usuarios no existe'
 	elif status_code == HTTP_METHOD_NOT_ALLOWED:
-		message = "You can't ask yourself for friendship. That's weird"
+		message = "No podes enviarte una solicitud de amistad a vos mismo. Eso es raro"
```

Comment:
That's weird

Reply:
Jajajaj me encantaba ese mensaje. Pero no se va a ver nunca, porque la app no te muestra a vos mismo para agregarte... mal fla